andela/travel_tool_front
# #160833193 Include Profile Drop Down To Navigation Bar

#### What does this PR do?
This PR ensure that users are able to see profile as part of the dropdown in navigation bar

#### Description of Task to be completed?
Profile dropdown should be populated in the navigation bar

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory and run git checkout `ft-profile-dropdown-160833193`
- Run the server `make start`
- Login to get started
- click on the drop-down icon close to the user's name
- click on profile, then you should be taken to the user's profile

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
[#160833193](https://www.pivotaltracker.com/story/show/160833193)

#### Screenshots (if appropriate)
<img width="1280" alt="screen shot 2018-10-01 at 11 57 30 pm" src="https://user-images.githubusercontent.com/26988297/46320354-cf371680-c5d5-11e8-83aa-a7262e9b513d.png">


#### Questions:
